From 0756d902734c3d4353264d1b4c2ccec87359bd4a Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Tue, 21 Apr 2015 17:26:56 +0200
Subject: [PATCH] tmpfiles: there's no systemd-forbid-user-logins.service
 service

(cherry picked from commit 451d691ae110a600497348d9f6288bc84efb8642)

Cherry-picked from: 0d2b365
Resolves: #1222517
---
 tmpfiles.d/systemd-nologin.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tmpfiles.d/systemd-nologin.conf b/tmpfiles.d/systemd-nologin.conf
index d61232b..a30a8da 100644
--- a/tmpfiles.d/systemd-nologin.conf
+++ b/tmpfiles.d/systemd-nologin.conf
@@ -5,7 +5,7 @@
 #  the Free Software Foundation; either version 2.1 of the License, or
 #  (at your option) any later version.
 
-# See tmpfiles.d(5) and systemd-forbid-user-logins.service(5).
+# See tmpfiles.d(5), systemd-user-session.service(5) and pam_nologin(8).
 # This file has special suffix so it is not run by mistake.
 
 F! /run/nologin 0644 - - - "System is booting up. See pam_nologin(8)"
